CHANGELOG.md

```diff
@@ -2,10 +2,9 @@
 
 ## v2.3.0
 
-* v2.2 introduced a [fix](https://github.com/mapbox/mapbox-navigation-ios/pull/3652) saying "It is a programmer error to have more than one alive `NavigationViewController`, `NavigationService` or `RouteController` simultaneously". Now you will receive a log message with fault level helping you to spot the issue. To pause debugger when SDK detect the problematic situation, you shall enable "All Runtime Issues" breakpoint in Xcode. Learn more about breakpoints in [Xcode documentation](https://developer.apple.com/documentation/xcode/setting-breakpoints-to-pause-your-running-app). ([#3740](https://github.com/mapbox/mapbox-navigation-ios/pull/3740))
-
 ### Packaging
 
+* v2.2.0 [clarified](https://github.com/mapbox/mapbox-navigation-ios/pull/3652) that is an error to have more than one instance of `NavigationViewController`, `NavigationService`, or `RouteController` running simultaneously. Now you will receive a log message at the fault level helping you to spot the issue. To pause the debugger when the SDK detect the problematic situation, enable the “All Runtime Issues” breakpoint in Xcode. Learn more about breakpoints in [Xcode documentation](https://developer.apple.com/documentation/xcode/setting-breakpoints-to-pause-your-running-app). ([#3740](https://github.com/mapbox/mapbox-navigation-ios/pull/3740))
```

Comment:
This is definitely not a packaging changelog entry. And I wanted it to be at the top as well.

Reply:
Sorry, that was my suggestion. It just looked awkward to have a non-bullet-point at the top of the release notes; it happened to look like the “Changes since” line was referring only to this blurb that began by talking about v2.2.0. At the same time, I did want to respect the intention to give this blurb more prominence than it would have under the “Location tracking” or “Other changes” section.

“Packaging” admittedly isn’t very accurate. Here in the changelog, it looks better above the “Packaging” section, so let’s move it back there. We can tweak the final release’s release notes to put this blurb at the very top, before “Other changes since”. The other changes to this blurb should remain, because the previous wording inadvertently came across as overly stern.